deps/v8/.gitignore

```diff
@@ -1,82 +0,0 @@
-*.a
```

Comment:
because `cp` doesn't copy `.*` from the dir. it can be reverted if it's an issue.